manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml

```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: OpenStreetMap.Josm
+Publisher: OpenStreetMap
+PackageName: JOSM
+PackageVersion: 17833
+License: CC-BY-SA and LGPL since April 2014
```

Comment:
JOSM is licensed GPL-2.0-or-later, see https://josm.openstreetmap.de/browser/josm/trunk/LICENSE